src/api/image/src/routes/image.js

```diff
@@ -22,8 +22,8 @@ router.use(
   celebrate({
```

Comment:
The comment above this line needs to be updated to include the fact that we now allow passing an integer too.

```
* We also support passing an image name as a param in the URL:
 *
 * - image: should look like look '_ok8uVzL2gI.jpg'. Don't allow filenames like '../../dangerous/path/traversal'.
 */
```